src/Cmts.mli

```diff
@@ -56,7 +56,7 @@ val relocate :
 val fmt_before :
      t
   -> Conf.t
-  -> fmt_code:(Conf.t -> string -> Fmt.t)
+  -> fmt_code:(string -> (Fmt.t, unit) Result.t)
```

Comment:
`Cmts.ml` have the `Conf.t` already so it makes sense to pass it to `fmt_code` here, to be sure the conf is uptodate and avoid having 2 ways to pass the same thing.

Reply:
Yes but I find this more consistent with Fmt_odoc. And it also avoids the conf to be modified somewhere in Cmts. (currently it is just passed around each function, but better be safe, it shouldn't be modified since the call to `Cmts.fmt_*` from Fmt_ast).

Reply:
Nothing prevent the config from changing inside `Cmts.fmt_*` and it's usually changing when used in `Fmt_ast`, we should not add implicit assumption.
I think we should pass it to `fmt_code`, for consistency with how `Cont.t` is used elsewhere.

Reply:
I reverted this commit